kafka-connect-s3/src/main/java/io/confluent/connect/s3/TopicPartitionWriter.java

```diff
@@ -396,32 +398,44 @@ private boolean rotateOnTime(String encodedPartition, Long recordTimestamp, long
     );
 
     log.trace(
-        "Checking rotation on time with recordCount '{}' and encodedPartition '{}'",
```

Comment:
added topic-partition to all these logs customer sent logs and without this it was not easy to figure out which log is for which partition as customer had many topics and many partitions on a connector.